src/commands/base-command.ts

```diff
@@ -606,7 +609,8 @@ export default class BaseCommand extends Command {
       token,
       ...apiUrlOpts,
     })
-    const { buildDir, config, configPath, repositoryRoot, siteInfo } = cachedConfig
+    const { buildDir, config, configPath, env, repositoryRoot, siteInfo } = cachedConfig
+    env.NETLIFY_CLI_VERSION = { sources: ['internal'], value: netlifyCliVersion }
```

Comment:
This is the main functional change in this PR.

I don't have strong feelings around the name of the var - maybe `NETLIFY_CLI` would be better? We already have `NETLIFY_LOCAL` that we set to `true`. We could also reuse that, but I could imagine that some consumers are depending on its value being `"true"`.

Reply:
I think this var name is just fine - clear is best imo (even if it's wordy :D )